plugins/qeta/src/components/QuestionsContainer/QuestionsContainer.tsx

```diff
@@ -87,6 +89,7 @@ export const QuestionsContainer = (props: QuestionsContainerProps) => {
 
   const onSearchQueryChange = (event: React.ChangeEvent<HTMLInputElement>) => {
     onPageChange(1);
+    analytics.captureEvent('qeta-search', event.target.value ?? '');
```

Comment:
```suggestion
    analytics.captureEvent('qeta-search', event.target.value ?? '');
```
I used _qeta-_ as a prefix so as not to confuse it with the native backstage search